### PR TITLE
feat(forms-web-app): confirm information to be submitted

### PR DESCRIPTION
**Pull request by a6software**
feat(forms-web-app): confirm information to be submitted

## Ticket Number

UCD-793

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] I have merged commits to avoid fixing things in this PR
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: forms-web-app/src/controllers/confirmation.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const VIEW = {
+  CONFIRMATION: 'confirmation/index',
+};
+
+exports.getConfirmation = (req, res) => {
+  const appellantEmail = req.session && req.session.appeal && req.session.appeal['appellant-email'];
+  // req.session.destroy();
```
Comment (MrSimonEmms) [inline forms-web-app/src/controllers/confirmation.js:7]: Remove commented-out code

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
LGTM, except the commented out code

File: forms-web-app/src/controllers/confirmation.js (hunk 1 above)
Comment (a6software) [reply forms-web-app/src/controllers/confirmation.js]: Done, cheers

File: forms-web-app/src/app.js [hunk 2]
```diff
@@ -73,7 +75,9 @@ app.use('/', indexRouter);
 app.use('/application-name', applicationNameRouter);
 app.use('/application-number', applicationNumberRouter);
 app.use('/check-answers', checkAnswersRouter);
+app.use('/confirmation', confirmationRouter);
 app.use('/eligibility', eligibilityRouter);
+app.use('/submission', submissionRouter);
```
Comment (Fenisay) [inline forms-web-app/src/app.js:80]: Is this not supposed to be appellant-submission ? Or it's another thing maybe

Comment (a6software) [reply forms-web-app/src/app.js:80]: I just copied what they gave on the ticket 

https://has-appeal-alpha.herokuapp.com/v5/submission

**Review by MrSimonEmms — APPROVED** [review]

**Comment by a6software** [comment]
ty :)

**Pull request merged**